opae.cfg

```diff
@@ -0,0 +1,459 @@
+              { "device": "a10gx_vf", "feature_id": "*" }
+            ]
+          }
+        ],
+        "fpgad": [],
+        "rsu": [] 
+      }
+    },
+
+    "d5005": {
+      "enabled": true,
+      "platform": "Intel FPGA Programmable Acceleration Card D5005",
+
+      "devices": [
+        { "name": "d5005_pf", "id": [ "0x8086", "0x0b2b", "*", "*" ] },
+        { "name": "d5005_vf", "id": [ "0x8086", "0x0b2c", "*", "*" ] }
+      ],
+
+      "opae": {
+        "plugin": [
+          {
+            "enabled": true,
+            "module": "libxfpga.so",
+            "devices": [ "d5005_pf", "d5005_vf" ],
+            "configuration": {}
+          }
+        ],
+        "fpgainfo": [
+          {
+            "enabled": true,
+            "module": "libboard_d5005.so",
+            "devices": [
+              { "device": "d5005_pf", "feature_id": "*" },
+              { "device": "d5005_vf", "feature_id": "*" }
+            ]
+          }
+        ],
+        "fpgad": [],
+        "rsu": []
+      }
+    },
+
+    "n3000": {
+      "enabled": true,
+      "platform": "Intel FPGA Programmable Acceleration Card N3000",
+
+      "devices": [
+        { "name": "n3000_pf", "id": [ "0x8086", "0x0b30", "*", "*" ] },
+        { "name": "n3000_vf", "id": [ "0x8086", "0x0b31", "*", "*" ] }
+      ],
+
+      "opae": {
+        "plugin": [
+          {
+            "enabled": true,
+            "module": "libxfpga.so",
+            "devices": [ "n3000_pf", "n3000_vf" ],
+            "configuration": {}
+          }
+        ],
+        "fpgainfo": [
+          {
+            "enabled": true,
+            "module": "libboard_n3000.so",
+            "devices": [
+              { "device": "n3000_pf", "feature_id": "*" },
+              { "device": "n3000_vf", "feature_id": "*" }
+            ]
+          }
+        ],
+        "fpgad": [
+          {
+            "enabled": true,
+            "module": "libfpgad-vc.so",
+            "devices": [ "n3000_pf" ],
+            "configuration": {
+              "cool-down": 30,
+              "get-aer":     [ "setpci -s %s ECAP_AER+0x08.L",
+                               "setpci -s %s ECAP_AER+0x14.L" ],
+              "disable-aer": [ "setpci -s %s ECAP_AER+0x08.L=0xffffffff",
+                               "setpci -s %s ECAP_AER+0x14.L=0xffffffff" ],
+              "set-aer":     [ "setpci -s %s ECAP_AER+0x08.L=0x%08x",
+                               "setpci -s %s ECAP_AER+0x14.L=0x%08x" ],
+              "sensor-overrides": [
+                {
+                  "enabled": true,
+                  "name": "12V AUX Voltage",
+                  "low-warn": 11.40,
+                  "low-fatal": 10.56
+                }
+              ]
+            }
+          }
+        ],
+        "rsu": []
+      }
+    },
+
+    "n5010": {
+      "enabled": true,
+      "platform": "Silicom FPGA SmartNIC N5010 Series",
+
+      "devices": [
+        { "name": "n5010_pf", "id": [ "0x1c2c", "0x1000", "0", "0" ] },
```

Comment:
["0x1c2c", "0x1000", "0", "0" ]    why "0"?  "*" may be correct ?
same as N3000 and D5005

Reply:
It was set to "0" by Roger here: https://github.com/OPAE/opae-sdk/pull/2654/files